src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -148,7 +149,8 @@ private void setupListView() {
 
     private void setupKeyboardShortcuts() {        
         addEventHandler(KeyEvent.KEY_PRESSED, event -> {
-            if (KeyboardShortcuts.markAsRead.match(event)) {
+            // Temporary fix for now since markAsRead and Show Related Issue/PR have same keys
+            if (KeyboardShortcuts.markAsRead.match(event) && !SHOW_RELATED_ISSUE_OR_PR.match(event)) {
```

Comment:
This confused me until I saw the next part. :stuck_out_tongue: Maybe you could add to the comment, saying it will only work if the key for marking as read is not the default.